lib/meaning-cloud.rb

```diff
@@ -17,10 +17,14 @@ def self.configure
 
   # Main configuration class.
   class Configuration
-    attr_accessor :key
+    attr_accessor :key,
+		  :language,
+		  :topic_types
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.